src/main/shared/player/player-model.ts

```diff
@@ -0,0 +1,10 @@
+import { IObject } from '../util/util-model';
+
+export interface Player extends IObject {
+   id: string;
+   name: string;
+   position: {
```

Comment:
need to store the direction the user is pointing the gun too!

Reply:
Sure, there will be a lot more property I just tarted with position. Didnt want to do everything in one run :)